### PR TITLE
git-diff-index, git-diff-tree: add page

### PR DESCRIPTION
**Pull request by ShamimShahraeini**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by ShamimShahraeini** [comment]
https://github.com/tldr-pages/tldr/issues/3953

**Review by sebastiaanspeck — CHANGES_REQUESTED** [review]

File: pages/common/git-diff-tree.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+`git diff-tree -r {{commit}} {{commit}}`
+
+- Display changes in patch format:
+
+`git diff-tree -p {{tree-ish}} {{tree-ish}}`
+
+- Filter changes by a specific path:
+
+`git diff-tree {{tree-ish}} {{tree-ish}} -- {{path/to/file_or_directory}}`
```
Comment (kbdharun) [inline pages/common/git-diff-tree.md:20]:
```suggestion
`git diff-tree {{tree-ish1}} {{tree-ish2}}`

- Show changes between two specific commits:

`git diff-tree -r {{commit1}} {{commit2}}`

- Display changes in patch format:

`git diff-tree -p {{tree-ish1}} {{tree-ish2}}`

- Filter changes by a specific path:

`git diff-tree {{tree-ish1}} {{tree-ish2}} -- {{path/to/file_or_directory}}`
```
A minor suggestion to distinguish the placeholders.

**Review by kbdharun — APPROVED** [review]
LGTM, Welcome to tldr and thanks for your contribution.

File: pages/common/git-diff-tree.md (hunk 1 above)
Comment (ShamimShahraeini) [reply pages/common/git-diff-tree.md:20]: oh, right! thank you

**Review by sebastiaanspeck — APPROVED** [review]

**Pull request merged**